src/frontend/services/UserService.jsx

```diff
@@ -0,0 +1,116 @@
+import { getAllUsers, getUserData } from "frontend/redux/Slices/UserSlice";
+import axios from "axios";
+import {
+  handleFollowUser,
+  updateUserData,
+} from "frontend/redux/Slices/AuthSlice";
+import { toggleLoader } from "frontend/redux/Slices/PostSlice";
+import { Notify } from "frontend/components";
+
+export const fetchAllUsers = () => {
+  return async (dispatch) => {
+    try {
+      const res = await axios.get("/api/users");
+      dispatch(getAllUsers(res.data.users));
+    } catch (error) {
+      console.log(error);
+    }
+  };
+};
+
+export const fetchUserData = (userId) => {
+  return async (dispatch) => {
+    try {
+      const res = await axios.get(`/api/users/${userId}`);
+      dispatch(getUserData(res.data.user));
+    } catch (error) {
+      console.log(error);
+    }
+  };
```

Comment:
In all catch statements, try to use `console.error`

Reply:
ok, ill change all catch statements to console.error in my next PR